packages/vm/examples/module_size.rs

```diff
@@ -0,0 +1,65 @@
+use std::fs::File;
+use std::io::Read;
+use std::mem;
+
+use clap::{App, Arg};
+
+use cosmwasm_vm::internals::compile_and_use;
+use cosmwasm_vm::internals::make_runtime_store;
+use cosmwasm_vm::Size;
+use wasmer::Module;
+
+pub fn main() {
+    let matches = App::new("Module size estimation")
+        .version("0.0.2")
+        .author("Mauro Lacy <mauro@lacy.com.es>")
+        .arg(
+            Arg::with_name("WASM")
+                .help("Wasm file to read and compile")
+                .required(true)
+                .index(1),
+        )
+        .get_matches();
+
+    // File
+    let path = matches.value_of("WASM").expect("Error parsing file name");
+    let mut file = File::open(path).unwrap();
+    mem::drop(matches);
+
+    // Read wasm
+    let mut wasm = Vec::<u8>::new();
+    file.read_to_end(&mut wasm).unwrap();
+    mem::drop(file);
+
+    // Report size
+    let wasm_size = wasm.len();
+    println!("wasm size: {} bytes", wasm_size);
+
+    // Compile module
+    let memory_limit = Some(Size::mebi(10));
+    let module = compile_and_use(&wasm, memory_limit).unwrap();
```

Comment:
What about wrapping this in a function too? Then it is more consistent and we don't run into issues if `compile_and_use` is used elsewhere in the script.